tasks/cephfs/test_volume_client.py

```diff
@@ -11,8 +11,8 @@
 
 class TestVolumeClient(CephFSTestCase):
     # One for looking at the global filesystem, one for being
-    # the VolumeClient, one for mounting the created shares
-    CLIENTS_REQUIRED = 3
+    # the VolumeClient, two for mounting the created shares
+    CLIENTS_REQUIRED = 4
```

Comment:
Yeah, did not completely understand how these hooks work. Did a union of the two above comments, and referred to https://github.com/ceph/ceph-qa-suite/commit/b06818cf4be2f822e5737bcebc089bf5dd8fc7d3 . Hope this works.